ci/repo-config/mesosci/slc7/o2-dataflow.env

```diff
@@ -5,7 +5,7 @@ ALIBUILD_DEFAULTS=o2-dataflow
 PR_REPO=alisw/alidist
 PR_BRANCH=master
 NO_ASSUME_CONSISTENT_EXTERNALS=
-TRUST_COLLABORATORS=true
+#% TRUST_COLLABORATORS true
```

Comment:
What is this for?

Reply:
Only `list-branch-pr` uses this, so I thought this would make it clear that `TRUST_COLLABORATORS` (and `TRUSTED_USERS`) is magic, to stop people from removing the variables because they aren't used by the shell scripts.

Reply:
@ktf Would you prefer it if I made those shell variables (i.e. `TRUST_COLLABORATORS=true` as before)?